EnnoKalbfell/autozon
# My products page

**Changes:**
- Added new page
- Used token in sessionStorage for authorization
- Fixed linter issues

**If you want to test it:**
_No products visible if not logged in:_
1. Go to localhost:4200/my-products
2. You should see no products

_Products visible if logged in:_
1. Go to localhost:4200/login and login with dealer
2. Go to localhost:4200/my-products
3. You should see products
